addon/components/frost-table.js

```diff
@@ -25,19 +29,42 @@ export default Component.extend({
     items: PropTypes.array.isRequired,
 
     // options
-    onCallback: PropTypes.func
+    itemKey: PropTypes.string,
+    selectedItems: PropTypes.arrayOf(PropTypes.oneOfType([
+      PropTypes.EmberObject,
+      PropTypes.object
+    ])),
+    onCallback: PropTypes.func,
+    onSelectionChange: PropTypes.func,
 
     // state
+    _isShiftDown: PropTypes.bool,
+    _itemComparator: PropTypes.func,
+
+    _rangeState: PropTypes.shape({
+      anchor: PropTypes.oneOfType([
+        PropTypes.EmberObject,
+        PropTypes.object
+      ]),
+      endpoint: PropTypes.oneOfType([
+        PropTypes.EmberObject,
+        PropTypes.object
+      ])
+    })
   },
 
   getDefaultProps () {
     return {
       // options
       columns: [],
       items: [],
```

Comment:
They're internal and functional so probably not necessary, but again doesn't hurt so sure